modules/050-network-policy-engine/images/kube-router/werf.inc.yaml

```diff
@@ -0,0 +1,57 @@
+{{- $binaries := "/sbin/xtables-nft-multi /sbin/ip6tables-nft /sbin/ip6tables-nft-restore /sbin/ip6tables-nft-save /sbin/iptables-nft /sbin/iptables-nft-restore /sbin/iptables-nft-save /sbin/xtables-legacy-multi /sbin/iptables /sbin/iptables-restore /sbin/iptables-save /sbin/iptables-legacy /sbin/iptables-legacy-restore /sbin/iptables-legacy-save /sbin/ip6tables /sbin/ip6tables-restore /sbin/ip6tables-save /sbin/ip6tables-legacy /sbin/ip6tables-legacy-restore /sbin/ip6tables-legacy-save /usr/lib64/libnetfilter_conntrack.so* /sbin/ipset /usr/bin/ip /usr/sbin/conntrack" }}
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-binaries-artifact
+fromImage: common/alt
+shell:
+  install:
+    - apt-get update
+    - apt-get install -y iptables iptables-nft ipset iproute2 conntrack-tools
+    - /binary_replace.sh -i "{{ $binaries }}" -o /relocate
+    - |
+      for cmd in iptables iptables-save iptables-restore ip6tables ip6tables-save ip6tables-restore ip6tables-nft ip6tables-nft-restore ip6tables-nft-save iptables-nft iptables-nft-restore iptables-nft-save; do
+        ln -f -s /iptables-wrapper "/relocate/sbin/${cmd}"
+      done
+      # broken symlinks are not imported from the artifact
+      touch /iptables-wrapper
+---
+artifact: {{ .ModuleName }}/kube-router-artifact
+from: {{ .Images.BASE_GOLANG_20_ALPINE }}
+shell:
+  install:
+    - apk add --no-cache git make
+    - mkdir -p /src
+    - cd /src
+    - git clone --branch v0.3.2 --depth 1 {{ $.SOURCE_REPO }}/cloudnativelabs/kube-router.git .
+    - |
+      cat <<EOF >> go.mod
+      module github.com/cloudnativelabs/kube-router
+
+      go 1.20
+      EOF
+    - GIT_COMMIT=$(git describe --tags --dirty)
+    - BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z)
+    - GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -mod vendor -ldflags "-X github.com/cloudnativelabs/kube-router/pkg/cmd.version=${GIT_COMMIT} -X github.com/cloudnativelabs/kube-router/pkg/cmd.buildDate=${BUILD_DATE}" -o kube-router cmd/kube-router/kube-router.go
+    - export GOPROXY={{ $.GOPROXY }}
+    - make kube-router
```

Comment:
Why we first build kube-router and than do make kube-router ?

Reply:
fixed